docs/next/guides/sidebar.js

```diff
@@ -199,6 +201,7 @@ module.exports = {
     { title: 'Formulas', children: formulasItems },
     { title: 'Accessories and Menus', children: accessoriesAndMenusItems },
     { title: 'Internationalization', children: internationalizationItems },
+    { title: 'Accessibility', children: accessibilityItems },
```

Comment:
Before merging, make suer that the Accessibility section was approved. There is an ongoing discussion: https://handsoncode.slack.com/archives/C026GBFHQBA/p1644017748705709

Reply:
Not approved: https://handsoncode.slack.com/archives/C026GBFHQBA/p1646300303853959?thread_ts=1644017748.705709&cid=C026GBFHQBA

Taking it back then